src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2888,8 +2888,8 @@ public void setStereoscopy(String stereoscopy) {
 		SBSL,
 		SBSR,
 		HSBSL,
```

Comment:
@valib `HSBSL` do not exist anymore, is that correct ?
https://github.com/FFmpeg/FFmpeg/blob/415f907ce8dcca87c9e7cfdc954b92df399d3d80/libavfilter/vf_stereo3d.c